### DISCUSSION TITLE
Zabbix

### DISCUSSION THREAD

**Discussion by lanaparadinha** [olahallengren/sql-server-maintenance-solution#456] [Ideas]
How impact to sincronize backup with zabbix?

**Comment by olahallengren** [comment]
There is currently no integration with Zabbix.

@lanaparadinha, could you describe how you would like it to work?

**Reply by lanaparadinha** [reply]
Only send zabbix_sender if backup run with sucessfull or fail.

exec xp_cmdshell '"<Path_of_zabbix_sender>" -z <server> -p <port> -s <Hostname> -k <Key_monitored> -o <Sucess/fail>'

I see log into table CommandLog, I believe that one more monitoring mode would be important.

**Reply by lanaparadinha** [reply]
any?
